src/modules/lifecycle.json

```diff
@@ -10,7 +10,7 @@
       "name": "ready",
       "tags": [
         {
-          "name": "calls-metrics"
+          "name": "exclude-from-sdk"
```

Comment:
For some reason I couldn't have both tags, calls-metrics and exclude-from-sdk. Seems like it was failing some jsonschema validation. But even so, calls-metrics is just meant for generating things in the SDK, so maybe this is not needed?

Reply:
I think the manual code you have covers the use case, and we don't need the extra tag.

There's a test for it, so if your unit tests passed, we're good.

But beware, the unit test fails like 1 in 10 times due to a race condition in jest/the test code.